src/Microsoft.AspNetCore.Rewrite/IISUrlRewriteOptionsExtensions.cs

```diff
@@ -65,5 +65,17 @@ public static RewriteOptions AddIISUrlRewrite(this RewriteOptions options, TextR
 
             return options;
         }
+
+        /// <summary>
+        /// Adds a rule to the current rules.
+        /// </summary>
+        /// <param name="options">The <see cref="RewriteOptions"/>.</param>
+        /// <param name="rule">A <see cref="IISUrlRewriteRule"/> to be added to the current rules.</param>
+        /// <returns>The Rewrite options.</returns>
+        public static RewriteOptions AddIISUrlRewrite(this RewriteOptions options, IISUrlRewriteRule rule)
```

Comment:
I was about to fix these two items but am not sure how to get runtime 1.2.0-beta-001289-00 now required by this commit: https://github.com/aspnet/KoreBuild/commit/27d74d6e4ef457ce8d195c2506161582064bbf41.

Running `build.cmd` doesn't seem to fetch it like it did originally for me. Here's what I see now:

```The specified framework 'Microsoft.NETCore.App', version '1.2.0-beta-001289-00' was not found.
  - Check application dependencies and target a framework version installed at:
      C:\Users\dpeden\AppData\Local\Microsoft\dotnet\shared\Microsoft.NETCore.App\
  - The following versions are installed:
      1.0.0
      1.1.0
      1.2.0-beta-001269-00
      1.2.0-beta-001285-00
  - Alternatively, install the framework version '1.2.0-beta-001289-00'.
```

I'll go ahead and push but I can't run tests without this framework. Any ideas on how to fix this?